addons/binding/org.openhab.binding.mysensors/src/main/java/org/openhab/binding/mysensors/handler/MySensorsBridgeHandler.java

```diff
@@ -0,0 +1,238 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mysensors.handler;
+
+import static org.openhab.binding.mysensors.MySensorsBindingConstants.*;
```

Comment:
For static imports I do not care too much. But afaik, the Eclipse code formatter anyhow replaces them by individual imports...